First review for translation of: Softwarereview_intro.pt.Rmd (#852)
ropensci/dev_guide: softwarereview_intro.pt.Rmd

> Você pode reconhecer os pacotes da rOpenSci que foram revisados por pares por meio de um selo (*badge*) verde com o texto "peer-reviewed" em seu README, com links para suas revisões ([por exemplo](https://github.com/ropensci/restez#locally-query-genbank-)); e por meio de um ícone de comentário azul próximo à sua descrição na [página de pacotes da rOpenSci](https://ropensci.org/packages/) com links para as revisões.

Author: It was quite difficult to understand what it meant in the translation.
I looked up on the website and this is an attempt to describe it a bit better.
But this is a change from the source book, I'm not sure how to proceed.

```suggestion
Você pode reconhecer os pacotes da rOpenSci que foram revisados por pares por meio de um selo (*badge*) verde com o texto "peer-reviewed" em seu README, com links para suas revisões ([por exemplo](https://github.com/ropensci/restez#locally-query-genbank-)); e por meio de um botão azul com o texto *"Peer-reviewed"* próximo à sua descrição na [página de pacotes da rOpenSci](https://ropensci.org/packages/) com links para as revisões.
```

Tecnicamente, aproveitamos ao máximo o [GitHub](https://github.com/): cada processo de revisão de pacote é uma *issue* no [repositório `ropensci/software-review` no GitHub](https://github.com/ropensci/software-review/). Por exemplo, clique em [aqui](https://github.com/ropensci/software-review/issues/24) para ler o tópico de avaliação do pacote `ropenaq`: o processo é uma conversa contínua até a aceitação do pacote, com duas revisões externas como marcos importantes. Além disso, usamos os recursos do GitHub, como o uso de *issue templates* (como modelos de envio) e uso de *labels* (etiquetas) que usamos para acompanhar o progresso dos envios (desde as verificações do(a) editor(a) até a aprovação).

> ## Por que enviar seu pacote para a rOpenSci? {#whysubmit}

Author: Revisei até aqui!

Reviewer: Ola @beatrizmilz is coming here to ask about the status of this review.  Do you think you will have the time to finished?  Thanks in advance!

Author: > Ola @beatrizmilz is coming here to ask about the status of this review. Do you think you will have the time to finished? Thanks in advance!

Hi Yani! I think I started it on the translaton and forgot to add on my to-do list!
I just added, I can finish it soon!

Além disso, os(as) autores(as) e revisores(as) podem entrar em contato com os(as) editores(as) em particular se tiverem alguma dúvida ou questão.

## Como os usuários saberão que um pacote foi revisado? {#how-will-users-know-a-package-has-been-reviewed}

- O README do seu pacote apresentará um selo de revisão por pares com um link para o tópico de revisão do software.
- Seu pacote terá um [site de documentação em `docs.ropensci.org`](#rodocsci) que você pode referenciar no DESCRIPTION.  
- Se você quiser, o repositório do seu pacote poderá ser transferido para a organização rOpenSci.

> - Se os(as) revisores(as) [concordarem em ser listados(as) no DESCRIPTION](#authorship), seus metadados mencionarão a revisão.

Author: The phrase is not 100% clear to me.
In "their metadata will mention the review.", "their metadata" is referring to the package metadata? 
If so, we can make if more explicit:
```suggestion
- Se os(as) revisores(as) [concordarem em ser listados(as) no DESCRIPTION](#authorship), os metadados do pacote mencionarão a revisão.
```

- O LEIAME do seu pacote apresentará um selo de revisão por pares com um link para o tópico de revisão do software.
- Seu pacote receberá um selo [`docs.ropensci.org` site de documentos](#rodocsci) que você pode vincular a partir da DESCRIÇÃO.

> - Se você quiser, o repositório do seu pacote poderá ser transferido para a organização rOpenSci.

Author: Making more clear what we mean by transfering to rOpenSci org. 
But this is not in the English version, check how to proceed.
```suggestion
- Se você quiser, o repositório do seu pacote poderá ser transferido para a organização rOpenSci no GitHub.
```

> - O repositório do seu pacote será transferido para a organização rOpenSci.

Reviewer 2: ```suggestion
- Se você quiser, o repositório de seu pacote será transferido para a organização rOpenSci.
```

@beatrizmilz this is the change made in #864, could we please add it for Portuguese?

source(file.path("scripts", "airtable-get-reviewers.R"), local = knitr::knit_global())
```

> ### Editores(as) Convidados(as) {#guesteditors}

Author: This is not in the original version (english). But to follow the structure, I added this h3-title. 

Let me know what do you think, Yani. We can also edit it in English and Spanish.
